api/rs/slint/mcu.md

```diff
@@ -80,42 +76,42 @@ in a format that's suitable for the software based renderer we're going to use.
 
 The operating system provides an event loop to connect and schedule these tasks. Slint implements the
-task of receiving user input and forwarding it to the user interface layer, as well as rendering the user interface to the screen.
+task of receiving user input and forwarding it to the user interface layer, and rendering the user interface to the screen.
 
-In bare metal environments it is your responsibility to substitute and connect functionality that is otherwise provided by the operating system:
+In bare metal environments it's your responsibility to substitute and connect functionality that's otherwise provided by the operating system:
 
- * Select crates that allow you to initialize the chips that operate peripherals, such as a touch input or display controller.
-   If there are no crates, you may have to to develop your own drivers.
- * Drive the event loop yourself by querying peripherals for input, forwarding that input into computational modules of your
+ * Select crates that initialize the chips that operate peripherals, such as a touch input or display controller.
+   If there are no crates, you may have to develop your own drivers.
+ * Drive the event loop by querying peripherals for input, forwarding that input into computational modules of your
    application and instructing Slint to render the user interface.
 
-In Slint, the two primary APIs you need to use to accomplish these tasks are the [`slint::platform::Platform`] trait as well as the [`slint::Window`] struct.
+In Slint, the two primary APIs you need are the [`slint::platform::Platform`] trait and the [`slint::Window`] struct.
 In the following sections we're going to cover how to use them and how they integrate into your event loop.
 
 ### The `Platform` Trait
 
 The [`slint::platform::Platform`] trait defines the interface between Slint and platform APIs typically provided by operating and windowing systems.
 
-You need to provide a minimal implementation of this trait and call [`slint::platform::set_platform`] before constructing your Slint application.
+You need to write a minimal implementation of this trait and call [`slint::platform::set_platform`] before constructing your Slint application.
 
 This minimal implementation needs to cover two functions:
 
- * `fn create_window_adapter(&self) -> Result<Rc<dyn WindowAdapter + 'static>, PlatformError>;`: Implement this function to return an implementation of the `WindowAdapter`
-   trait that will be associated with the Slint components you create. We provide a convenience struct [`slint::platform::software_renderer::MinimalSoftwareWindow`]
+ * `fn create_window_adapter(&self) -> Result<Rc<dyn WindowAdapter + 'static>, PlatformError>;`: This function returns an implementation of the `WindowAdapter`
+   trait that will be associated with the Slint components you create. Slint has a convenience struct [`slint::platform::software_renderer::MinimalSoftwareWindow`]
```

Comment:
Why did you change this from actionable ("Implement this") to passive? The context here is a to instruct the developer what they have to do. In API documentation of a function that the user can call, I'd use the term "This function returns XYZ", but the context here is a trait where a function needs to be implemented. The user will never call `create_window_adapter`.